test/e2e_node/mirror_pod_grace_period_test.go

```diff
@@ -100,6 +100,35 @@ var _ = SIGDescribe("MirrorPodWithGracePeriod", func() {
 			framework.ExpectEqual(pod.Spec.Containers[0].Image, image)
 		})
 
+		ginkgo.It("should update a static pod when the static pod is updated multiple times during the graceful termination period [NodeConformance]", func() {
+			ginkgo.By("get mirror pod uid")
+			pod, err := f.ClientSet.CoreV1().Pods(ns).Get(context.TODO(), mirrorPodName, metav1.GetOptions{})
+			framework.ExpectNoError(err)
+			uid := pod.UID
+
+			ginkgo.By("update the pod manifest multiple times during the graceful termination period")
+			for i := 0; i < 300; i++ {
+				err = createStaticPod(podPath, staticPodName, ns,
```

Comment:
If you don't create a pod with a fixed UID, then every update is going to have to wait until the old pod is completely cleaned up before the new one is started (it's the same as waiting in the apiserver for the pod to be fully deleted before another one can be created).  This test doesn't seem to make sense to me - because the UID isn't static and so no mirror pod update is possible (the UID should be changed).

Reply:
Although I may have incorrectly `checkMirrorPodRecreatedAndRunning` - which takes the UID and makes sure it is different.

If so, then this test does seem to be showing "updates to the same static pod name/namespace with different uids repeatedly don't converge".